python/graphstorm/gconstruct/remap_result.py

```diff
@@ -49,8 +66,62 @@
 # id_maps to each worker process.
 id_maps = {}
 
+def write_data_parquet_file(data, file_prefix, col_name_map=None):
```

Comment:
i think i have these functions in gconstruct. should we merge them?

Reply:
This function calls the write_data_parquet in gconstruct.